frontend/javascripts/oxalis/view/right-border-tabs/tree_hierarchy_view.tsx

```diff
@@ -254,11 +253,11 @@ function TreeHierarchyView(props: Props) {
   const checkedKeys = deepFlatFilter(UITreeData, (node) => node.isChecked).map((node) => node.key);
 
   // selectedKeys is mainly used for highlighting, i.e. blueish background color
-  const selectedKeys = props.selectedTreeIds.map((treeId) =>
-    getNodeKey(GroupTypeEnum.TREE, treeId),
-  );
+  let selectedKeys = props.selectedTreeIds.map((treeId) => getNodeKey(GroupTypeEnum.TREE, treeId));
+
+  if (props.activeGroupId) selectedKeys = [getNodeKey(GroupTypeEnum.GROUP, props.activeGroupId)];
 
-  if (props.activeGroupId) selectedKeys.push(getNodeKey(GroupTypeEnum.GROUP, props.activeGroupId));
+  treeRef.current?.scrollTo({ key: selectedKeys[0], align: "auto" });
```

Comment:
The current state of the code can be used to trigger the unwanted behavior:

- create a lot of trees
- give two trees at the top of the list a specific name
- search for that name and select it via the select all
- scroll down so that those trees are outside the rendered part of the tree list
- open the context menu on one of the bottom trees. 
=> this will trigger an auto scroll to the top of the list which is unwanted and the context menu no longer hovers the skeleton on which the context menu was initially opened

Reply:
good point!

Reply:
I think my changes solved this :)